src/Compilers/CSharp/Portable/FlowAnalysis/CSharpDataFlowAnalysis.cs

```diff
@@ -296,6 +298,33 @@ public sealed override bool Succeeded
             }
         }
 
+        internal TypeSymbolWithAnnotations TypeAndNullability
```

Comment:
>internal TypeSymbolWithAnnotations TypeAndNullability [](start = 8, length = 53)

I think the test hook can invoke NullableWalker.Analyze directly rather than adding this unrelated functionality into CSharpDataFlowAnalysis.